src/pretix/static/pretixpresale/scss/waiting.scss

```diff
@@ -18,3 +18,16 @@ body {
     font-size: 200px;
     color: $brand-primary;
 }
+
+.progress {
+    max-width: 450px;
+    margin: 0 auto 20px;
+}
+.steps {
+    max-width: 450px;
+    text-align: left;
+    margin: 0 auto 20px;
+}
+@for $i from 0 through 100 {
```

Comment:
Not sure we need that fine granularity – how fine will the task report the percentage anyway? Maybe just every 5 or even 10 percent steps? On the other hand, this will probably be well compressed with gzip, so it should not matter much adding 100 lines of CSS.

Reply:
I'd rather have more granularity here than less, for some really slow tasks even sub-percent progress is visible.